custom_components/steam_wishlist/translations/pt-BR.json

```
@@ -13,8 +13,17 @@
           },
           "description": "Digite o nome da sua conta Steam OU o ID do seu perfil Steam.",
           "title": "Configurar"
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Opções",
+        "data": {
+          "show_all_wishlist_items": "Exiba também itens da lista de desejos não em promoção no Upcoming Media Card (reinicialização necessária)"
         }
       }
     }
   }
```

You are missing the closing parenthesis: `Error: R] [JSON] Invalid JSON file translations/pt-BR.json` (https://github.com/boralyl/steam-wishlist/actions/runs/8415382357/job/23076727826?pr=20#step:3:224)

```suggestion
  }
}
```